pages/[id].js

```diff
@@ -39,16 +48,19 @@ export default function Detail() {
   }
 
   return (
-    <RoastDetailCard
-      onFavorite={addToFavorites}
-      name={data.name}
-      roaster={data.roaster}
-      arabica={data.arabica}
-      robusta={data.robusta}
-      level={data.level}
-      provenance={data.provenance}
-      score={data.score.map(({ rating }) => rating)}
-      session={session}
-    />
+    <>
+      <RoastDetailCard
+        onFavorite={addToFavorites}
+        name={data.name}
+        roaster={data.roaster}
+        arabica={data.arabica}
+        robusta={data.robusta}
+        level={data.level}
+        provenance={data.provenance}
+        score={data.score.map(({ rating }) => rating)}
```

Comment:
Not sure what the mapping does since you're not changing the contents of the `score` array. You map every entry to itself.

Wouldn't this work?
```suggestion
        score={data.score}
```

Reply:
I need to map because every entry in score consists of the userID and the actual score. So no user can rate the same roast twice.